docs/core/logger.md

```diff
@@ -49,9 +49,9 @@ These settings will be used across all logs emitted:
 | Setting                 | Description                                                                                                      | Environment variable            | Default Value       | Allowed Values                 | Example Value      | Constructor parameter |
 |-------------------------|------------------------------------------------------------------------------------------------------------------|---------------------------------|---------------------|--------------------------------|--------------------|-----------------------|
 | **Service name**        | Sets the name of service of which the Lambda function is part of, that will be present across all log statements | `POWERTOOLS_SERVICE_NAME`       | `service_undefined` | Any string                     | `serverlessAirline`| `serviceName`         |
-| **Logging level**       | Sets how verbose Logger should be                                                                                | `LOG_LEVEL`                     | `info`              |`DEBUG`, `INFO`, `WARN`, `ERROR`| `ERROR`            | `logLevel`            |
-| **Log incoming event**  | Whether to log or not the incoming event when using the decorator or middleware.                                 | `POWERTOOLS_LOGGER_LOG_EVENT`   | `false`             | `true`, `false`                | `false`            | `logEvent`            |
-| **Debug log sampling**  | Probability that a Lambda invocation will print all the log items regardless of the log level setting.           | `POWERTOOLS_LOGGER_SAMPLE_RATE` | `0`                 | `0.0` to `1`                   | `0.5`              | `sampleRateValue`     |
+| **Logging level**       | Sets how verbose Logger should be, from the most verbose to the least verbose (no logs)                                                                                | `LOG_LEVEL`                     | `info`              |`DEBUG`, `INFO`, `WARN`, `ERROR`, `SILENT` | `ERROR`            | `logLevel`            |
```

Comment:
nitpick: could you please apply formatting to the table so that the `|` are aligned?

Reply:
Sure. I thought there no order in these tables, because they are always looked like a mess in my vscode. I made the vscode window 6-x wider, and I see how it works 🙃

Reply:
Thank you for addressing my comment!

Should probably add some proper linting to markdown files at some point, you're right.

In the meantime I use this VSCode extension [that formats on save](https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one).